framework/src/engine/consensus/certificate_generation/utils.ts

```diff
@@ -76,6 +79,15 @@ export const verifyAggregateCertificateSignature = (
 		return false;
 	}
 
+	const validatorKeysWithWeights = [];
+	for (const validator of validators) {
+		validatorKeysWithWeights.push({
+			weight: validator.bftWeight,
+			blsKey: validator.blsKey,
+		});
+	}
+	const { weights, validatorKeys } = getSortedWeightsAndValidatorKeys(validatorKeysWithWeights);
```

Comment:
this util function seems to be used only in this function, so probably input name can be modified to take `Validator[]` as input so that it doesn't need to change the key name